devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -162,6 +162,7 @@ class HumanStateProvider::impl
     FloatingBaseName floatingBaseFrame;
 
     std::vector<std::string> jointList;
+    std::vector<double> CalibrationConfig;
```

Comment:
Accordingly to the other variables, it would be better to use small capital `calibrationConfig`.
Probably we could also use a name that specify that we are talking about joint configuration (e.g. `calibrationJointConfiguration`

Reply:
Ok